rotary/rules/polish.smk

```diff
@@ -122,61 +120,53 @@ rule polish_polypolish:
         """
 
 
-# TODO - the relative path workarounds in the shell here are a bit odd because polca outputs files in the present working directory
-rule polish_polca:
+rule polish_pypolca:
     input:
         qc_short_r1 = "{sample}/qc/{sample}_qc_R1.fastq.gz",
         qc_short_r2 = "{sample}/qc/{sample}_qc_R2.fastq.gz",
         polished = "{sample}/polish/polypolish/{sample}_polypolish.fasta"
     output:
-        polca_output = temp("{sample}/polish/polca/{sample}_polca.fasta"),
-        polypolish_sam = temp("{sample}/polish/polca/{sample}_polypolish.fasta.unSorted.sam"),
-        polypolish_bam = temp("{sample}/polish/polca/{sample}_polypolish.fasta.alignSorted.bam"),
-        read_mapping_files= temp(multiext("{sample}/polish/polca/{sample}_polypolish.fasta.bwa",
```

Comment:
@jmtsuji Does pyploca clean up its BWA files after it runs? If not we should leave some of these removed parameters in place so Snakemake deletes the BWA output files.

Reply:
Indeed, pypolca cleans up its BWA files -- thanks for checking!